docs/low-code-sql/development/gems/Transformations/aggregate.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Aggregate
+id: sql-aggregate
+description: Perform aggregation transformations with the option to group by columns.
+sidebar_position: 3
+tags:
+  - aggregate
+  - groupby
+  - data
+  - gem
+  - transformation
+---
+
+Together let's build an Aggregate Gem. Follow along in the `HelloWorld_SQL` Project.
+
+![1](../img/Snow4.6.1_Aggregate.png)
+
+1. **Open** the HelloWorld_SQL [Project](https://app.prophecy.io/metadata).
+2. From the list of Models, select the **Orders** Model. A Model is a series of transformation steps (Gems) that describe how to create a single table or view. The Orders Model defines the steps to create the Orders table.
+3. Open the **Transformation** dropdown to see the available Transformation Gems. The Aggregate Gem has already been dragged to the canvas and configured in this HelloWorld_SQL example.
+
+Let's explore the Aggregate Gem by checking the input and output data samples, and then check how the Aggregate Gem is configured.
+
+4. Click the arrow to **Run upto the Aggregate Gem**.
+5. Preview a **data sample** before the Aggregate Gem.
+6. This is the `payments` data sample before the Aggregate Gem. There is one row per payment.
+7. Click to **Run upto the Join Gem**.
+8. Preview a **data sample** after the Aggregate Gem.
+9. This is the `order_payments` data sample after the Aggregate Gem. The individual rows of `payments` have been grouped according to `ORDER_ID`, and the amounts have been summed according to the payment type.
+10. Let's see how the Aggregate Gem is configured. Click to **Open**.
+
+![2](../img/Snow4.6.2_Aggregate.png)
+
+1. There is one **Input** Dataset, `payments`, and we can see the columns and datatypes below.
+2. Open the **Group By** tab.
+3. We can see the Gem is configured to group according to the `order_id` column. Just click any column name listed in **(1) Input** to add a column to the GroupBy expressions.
```

Comment:
Just checking if it's "GroupBy" or "Group By"

Reply:
GroupBy
fixed!